.travis.yml

```diff
@@ -0,0 +1,31 @@
+sudo: false
+dist: trusty
+language: erlang
+otp_release:
+  - 20.0.1
+  - 19.3.6.1
+  - 18.3
+addons:
+  apt:
+    sources:
+      - debian-sid
+    packages:
+      ## GNU Make 4.  GNU Make 3.81 as [in Ubuntu 14.04
+      ## "trusty"](https://packages.ubuntu.com/trusty/make) is not
+      ## sufficient, because of error on `triq` test dep when running
+      ## `make tests`.  Also erlang.mk reports the following warning:
+      ## "erlang.mk:24: Please upgrade to GNU Make 4 or later:
+      ## https://erlang.mk/guide/installation.html".
+      - make
+before_install:
+  - make -v
+script:
+  - make ${JOB:?}
+matrix:
+  include:
+    - language: erlang
+      otp_release: 20.0.1
+      env: JOB=edoc
+env:
+  - JOB=tests
+  - JOB=dialyze
```

Comment:
README doesn’t mention this, but `make tests` and `make dialyze` can be combined into `make check`.

Reply:
Using `check` would mean there would be less jobs (hence less focused) and of longer duration. Do you prefer me to combine in one job only?

Reply:
If I understand correctly, less jobs means less compilation steps and less containers created, hence less total duration of jobs, so combining them should be beneficial.